vrs/marten-construction-script.py

```diff
@@ -106,12 +97,13 @@ def main(args):
     # for 1000 variants, 40 will be on chromosome X and only 1.5 will be on chromosome Y
 
     # Downsample the matrix table if the User doesn't have the capacity to annotate and go through 759million variants
-    if args.downsample is float and args.downsample < 1.00:
+    # if args.downsample is float and args.downsample < 1.00:
+    if args.downsample < 1.00:
```

Comment:
should add an else statement and raise a value error if the value is above 1.00 to help user realize the values needs to be below 1.00, see examples:
https://github.com/broadinstitute/gnomad_qc/blob/768d249cf02e1bc74614086d1fe098d69a622f01/gnomad_qc/v4/sample_qc/sex_inference.py#L316 

https://github.com/broadinstitute/gnomad_qc/blob/768d249cf02e1bc74614086d1fe098d69a622f01/gnomad_qc/v2/sample_qc/assign_subpops.py#L72

Reply:
ValueError included in coming commit